platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -907,12 +919,10 @@ else
     "$APP_NAME"_check
 fi
 
-base_message="Azure IoT Compatibility Script had $FAILURES Errors $WARNINGS Warnings $PASS Successful Checks and $SKIP Skipped Checks"
-if [ $FAILURES -gt 0 ]; then
-    wrap_bad "$base_message"
-    exit 1
-elif [ $WARNINGS -gt 0 ]; then
-    wrap_warning_message "$base_message"
-else
-    wrap_pass "$base_message"
-fi
+echo "----------------------------"
+echo "Compatibility Check Results"
+echo "----------------------------"
+echo "$(wrap_color "$PASS check(s) suceeded" green)"
+echo "$(wrap_color "$FAILURES check(s) failed" red)"
+echo "$(wrap_color "$WARNINGS check(s) have warnings" yellow)"
+echo "$(wrap_color "$SKIP check(s)" white)"
```

Comment:
I think we should add another message "run ./aziot-compatibility.sh --verbose for more details" . Same as iotedge check